launcher/src/main/java/org/apache/spark/launcher/SparkAppHandle.java

```diff
@@ -19,7 +19,7 @@
 
 /**
  * A handle to a running Spark application.
- * <p/>
+ * <p>
```

Comment:
I would just remove this entirely. Now, the tag isn't closed. The newline isn't important here anyhow. Feel free to clean up any other javadoc warnings you encounter.